CHANGELOG.md

```diff
@@ -97,6 +97,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add a setting to control auto release of OpenSearch managed index creation block ([#6277](https://github.com/opensearch-project/OpenSearch/pull/6277))
 - Fix timeout error when adding a document to an index with extension running ([#6275](https://github.com/opensearch-project/OpenSearch/pull/6275))
 - Clean up temporary files created during segment merge incase segment merge fails ([#6324](https://github.com/opensearch-project/OpenSearch/pull/6324))
+- Create writeable implementations for parser([#6364](https://github.com/opensearch-project/OpenSearch/pull/5838))
```

Comment:
Let's skip the changelog for this change

Reply:
what do you mean on skip? Are we going to merge another PR first? or just remove this changelog?

Reply:
Remove this changelog and add skip changelog label

Reply:
you mean add another label call Skip and move this changelog under skip label?

Reply:
You can delete this entry in the changelog and add the skip-changelog label to your PR

Reply:
got it, thanks